charmhelpers/fetch/ubuntu.py

```diff
@@ -242,8 +242,20 @@ def apt_cache(*_, **__):
     return ubuntu_apt_pkg.Cache()
 
 
-def apt_install(packages, options=None, fatal=False):
-    """Install one or more packages."""
+def apt_install(packages, options=None, fatal=False, override_env=None):
```

Comment:
I'm not sure about the `override_env=None` here.  My rationale is that I think that a function like `apt_install` should *by default* use a sanitized environment, and not expect the caller to work out what environment is safe for `apt_install` to work in.

I think this is true for all the functions in this module, that have been modified with the `override_env` module.  I would suspect that the cleaned environment of `/etc/environment`'s path should be enough to ensure `apt_*` functions work?

Reply:
Ok, let's do it, we're well into changing the world this cycle anyway.